cmd/vicadmin/server.go

```diff
@@ -287,12 +287,13 @@ func (s *server) loginPage(res http.ResponseWriter, req *http.Request) {
 		vs, err := vSphereSessionGet(&userconfig)
 		if err != nil || vs == nil {
 			// something went wrong or we could not authenticate
-			log.Warnf("User %s from %s failed to authenticate at %s", userconfig.User, req.RemoteAddr, time.Now())
+			log.Warnf("%s failed to authenticate at %s", req.RemoteAddr, time.Now())
 			http.Error(res, "Authentication failed due to incorrect credential(s)", http.StatusUnauthorized)
 			return
 		}
 
-		// successful login above; user is authenticated
+		// successful login above; user is authenticated, reported for audit purposed
+		log.Infof("User %s from %s was successfully to authenticated at %s", userconfig.User, req.RemoteAddr, time.Now())
```

Comment:
`userconfig.User` contains the username and password from the HTML form and we shouldn't be logging it this way, specially on a successful login.